### PR TITLE
refactor: change the way to call replication_ddl_client::set_app_envs

### PR DESCRIPTION
**Pull request by levy5307**
### What problem does this PR solve? 
Change the way to call set_app_replication_ddl_client::set_app_envs, becasue the return of this method is changed

### What is changed and how it works?
the calling of set_app_replication_ddl_client::set_app_envs is changed

### Check List 

Tests 

- Unit test
yes
- Integration test
no
- Manual test (add detailed scripts or steps below)
no
- No code
no

Code changes

- Has exported function/method change
no 
- Has exported variable/fields change
no
- Has interface methods change
no
- Has persistent data change
no

Side effects

- Possible performance regression
no
- Increased code complexity
no
- Breaking backward compatibility
no
Related changes

- Need to cherry-pick to the release branch
no
- Need to update the documentation
no
- Need to be included in the release note
no

### CODE REVIEW

File: src/shell/commands/table_management.cpp [hunk 1]
```diff
@@ -744,11 +744,18 @@ bool set_app_envs(command_executor *e, shell_context *sc, arguments args)
         values.emplace_back(args.argv[idx++]);
     }
 
-    ::dsn::error_code ret = sc->ddl_client->set_app_envs(sc->current_app_name, keys, values);
-
-    if (ret != ::dsn::ERR_OK) {
-        fprintf(stderr, "set app env failed with err = %s\n", ret.to_string());
+    auto response = sc->ddl_client->set_app_envs(sc->current_app_name, keys, values);
```
Comment (neverchanje) [inline src/shell/commands/table_management.cpp]:
```cpp
auto err_resp = sc->ddl_client->set_app_envs(sc->current_app_name, keys, values);
dsn::error_s err;
if(err_resp.is_ok()){
  err = err_resp.get_error();
}
if (!err.is_ok()) {
  fmt::print(stderr, "set app envs failed with rpc error {}!\n", err);
}
```

File: src/shell/commands/table_management.cpp [hunk 2]
```diff
@@ -744,11 +744,19 @@ bool set_app_envs(command_executor *e, shell_context *sc, arguments args)
...
     }
 
-    ::dsn::error_code ret = sc->ddl_client->set_app_envs(sc->current_app_name, keys, values);
-
-    if (ret != ::dsn::ERR_OK) {
-        fprintf(stderr, "set app env failed with err = %s\n", ret.to_string());
+    auto err_resp = sc->ddl_client->set_app_envs(sc->current_app_name, keys, values);
+    dsn::error_s err = err_resp.get_error();
+    std::string hint_msg;
+    if (err.is_ok()) {
```
Comment (vagetablechicken) [inline src/shell/commands/table_management.cpp]: err is ok, 这个听起来太奇怪了. 建议改成err_code之类的. 而且response的error和response的信息里的error, 可以从命名上区分的更清楚些

Comment (neverchanje) [reply src/shell/commands/table_management.cpp]: 要改就改 error_s 的名字。这个在 rocksdb 叫 Status::isOK，在 rust 叫 Result::is_ok，最后其实就是一个写法的问题。

Comment (vagetablechicken) [reply src/shell/commands/table_management.cpp]: 现在就专注于这里的表述就好了, 变量名改好就足够了. is_ok()这个没有什么问题, 我上面没表述清楚

Comment (levy5307) [reply src/shell/commands/table_management.cpp]: 我也觉得改就改errors_s的名字最好，单纯从类型和变量名来看，err这个变量起的其实没什么问题，你上面说的err is ok奇怪说起来还是error_s::is_ok()的问题，改变量也改变不了什么

**Pull request merged**